app/components/e-form-notification.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div :class="b(modifiers)">
+    <slot></slot>
```

Comment:
- [ ] Please add slot documentation for styleguide: https://github.com/vue-styleguidist/vue-styleguidist/blob/master/docs/Documenting.md#slots-documentation